### PR TITLE
Update release leads for 0.24 -> dprotaso,matzew

### PR DESCRIPTION
**Pull request by vaikas**
As per: https://github.com/knative/release#schedule

Update the release leads for the 0.24 release. Please ack the handoff for the next release:
@dprotaso 
@matzew

### CODE REVIEW

File: peribolos/knative-sandbox.yaml [hunk 1]
```diff
@@ -655,9 +655,8 @@ orgs:
           Knative Release Leads:
             description: Active member of the Knative Release Leads rotation.
             maintainers:
-            - tcnghia
-            - vaikas
-            - fallback-notification-blocker
```
Comment (markusthoemmes) [inline peribolos/knative-sandbox.yaml:660]: I think this should stay in until we remove all CODEOWNERS things.

Comment (vaikas) [reply peribolos/knative-sandbox.yaml:660]: lol, I just wanted to make it consistent :) I guess I went the wrong way :) 
https://github.com/knative/community/blob/main/peribolos/knative.yaml#L873

**Comment by dprotaso** [comment]
woott!

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

**Pull request merged**